### PR TITLE
Profilepage jp

### PR DESCRIPTION
**Pull request by jpcas**

### CODE REVIEW

**Review by tracilevine — APPROVED** [review]

**Review by DNiev — APPROVED** [review]

File: client/src/components/General.js [hunk 1]
```diff
@@ -1,37 +1,65 @@
...
+  };
+
+  const handleUpdate = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.patch('/api/user/me', formData);
+
+      sessionStorage.setItem('user', response.data);
+
+      setCurrentUser(response);
```
Comment (e-a-w) [inline client/src/components/General.js:23]: I believe this should also be response.data, as it is on line 21?

File: client/src/pages/Signup.js [hunk 2]
```diff
@@ -23,9 +23,15 @@ const SignUp = ({ history }) => {
       sessionStorage.setItem('user', response.data);
       setCurrentUser(response.data);
       if (response.data.admin) {
+<<<<<<< HEAD
```
Comment (e-a-w) [inline client/src/pages/Signup.js:26]: Looks like there's text leftover from a merge conflict here; you should choose either lines 27-29 or 31-33 but not both, and be sure to delete the text on 26, 30 and 34

File: client/src/pages/profile.js [hunk 3]
```diff
@@ -1,18 +1,59 @@
...
 import { Container, Image, Card } from 'react-bootstrap';
 import SideBar from '../components/SideBar';
 import Chat from '../components/Chat';
 import '../components/styles/profile.css';
 import FileUploader from '../components/Fileuploader';
+import axios from 'axios';
+import swal from 'sweetalert';
+import { AppContext } from '../context/AppContext';
+
+const Profile = ({ history: { push } }) => {
```
Comment (e-a-w) [inline client/src/pages/profile.js:11]: Two little things: first, make sure your file name is capitalized if it's a React component. Second, that {history: { push } } inside your parameters; is that working the way you expect it to? You should be able to just destructure {history} and then use history.push("/route"); in your code.

File: server/routes/open/index.js [hunk 4]
```diff
@@ -14,7 +14,10 @@ router.get('/password', requestPasswordReset);
 router.get('/password/:token', passwordRedirect);
 router.get('/bikeshop', getAllBikeshops);
 router.get('/bikeshop/find/:id', getBikeshopById);
+<<<<<<< HEAD
 router.get('/events', getAllEvents);
 router.post('/events', createEvent);
+=======
```
Comment (e-a-w) [inline server/routes/open/index.js:20]: Make sure you don't have extra text in the code leftover from merge conflicts.

File: yarn.lock [hunk 5]
```diff
@@ -489,6 +489,11 @@ axios@^0.19.2:
   dependencies:
     follow-redirects "1.5.10"
 
+<<<<<<< HEAD
+=======
+<<<<<<< HEAD
```
Comment (e-a-w) [inline yarn.lock:494]: Make sure you don't have extra text leftover from merge conflicts.

**Review by e-a-w — APPROVED** [review]

**Pull request merged**